content/src/Environment.ts

```diff
@@ -10,6 +10,8 @@ export const CURRENT_CONTENT_VERSION: EntityVersion = EntityVersion.V3
 const DEFAULT_STORAGE_ROOT_FOLDER = 'storage'
 const DEFAULT_SERVER_PORT = 6969
 const DEFAULT_DENYLIST_FILE_NAME = 'denylist.txt'
+const DEFAULT_DENYLIST_URLS = 'https://config.decentraland.org/denylist'
```

Comment:
can we make this a configurable environment var?

Reply:
If urlS are expected, shouldn't this be an array? Or what is the format for the variable?

Reply:
When the `Environment` is built it reads the `process.env.DENYLIST_URLS` env var, if not exists, it takes the default. Or you are saying of doing it differently?
The format is a string of urls separated by `space|newline|tab`, as it would also be configurable by an envvar.